techniques/system/common/1.0/cf-served.st

```diff
@@ -98,15 +98,12 @@ bundle common def
         string  => translatepath("${sys.workdir}/policy_server.dat"),
         comment => "Path to file containing address to policy server";
 
-   # Use localhost as default policy_server if no policy_server.dat exists
-    policy_server_set::
+      # Use "rudder" as default policy_server if no policy_server.dat exists
       "policy_server"
-        string  => readfile("${policy_server_file}", 2048),
+        string  => ifelse(fileexists("${def.policy_server_file}"),
+                          readfile("${policy_server_file}", 2048),
+                          "rudder"),
```

Comment:
this does not work correctly on Rudder Agent 2.10
If the file is not there, the output of the cf-serverd is
2016-01-12T11:43:47+0100  verbose: Path '/var/rudder/cfengine-community/bin/cf-agent' (encrypt=0)
2016-01-12T11:43:47+0100  verbose: Admit: '�P	`kQ	' root=
2016-01-12T11:43:47+0100  verbose: Admit: '��O	��K	' root=

....

2016-01-12T11:43:47+0100  verbose: IP '${def.policy_server}'
2016-01-12T11:43:47+0100  verbose: IP '�0N	�nQ	'
2016-01-12T11:43:47+0100  verbose: IP 'PK	��R	'